apps/meteor/app/livechat/server/lib/Contacts.ts

```diff
@@ -200,6 +210,40 @@ export async function createContact(params: CreateContactParams): Promise<string
 	return insertedId;
 }
 
+export async function updateContact(params: UpdateContactParams): Promise<ILivechatContact> {
+	const { contactId, name, emails, phones, customFields, contactManager, channels } = params;
+
+	const contact = await LivechatContacts.findOneById(contactId, { projection: { _id: 1 } });
+
+	if (!contact) {
+		throw new Error('error-contact-not-found');
+	}
+
+	if (contactManager) {
+		await validateContactManager(contactManager);
+	}
+
+	if (customFields) {
+		await validateCustomFields(customFields);
+	}
+
+	const updateContact: { $set: MatchKeysAndValues<ILivechatContact> } = {
+		$set: {
+			name,
+			emails,
+			phones,
+			contactManager,
+			channels,
+			customFields,
+			unknown: true,
+		},
+	};
+
+	await LivechatContacts.updateOne({ _id: contactId }, updateContact);
```

Comment:
It would be better to have a specific model method to do that instead of using updateOne here. WDYT?